test/fluentd-forward.sh

```diff
@@ -16,7 +16,7 @@ update_current_fluentd() {
 
     # undeploy fluentd
     os::log::debug "$( oc label node --all logging-infra-fluentd- )"
-    os::cmd::try_until_text "oc get daemonset logging-fluentd -o jsonpath='{ .status.numberReady }'" "0" $FLUENTD_WAIT_TIME
+    os::cmd::try_until_text "oc get daemonset -n logging logging-fluentd -o jsonpath='{ .status.numberReady }'" "0" $FLUENTD_WAIT_TIME
```

Comment:
There are more "oc get daemonset logging-fluentd ..." command lines (20 of them in total) in the test directory.  Do we want to add "-n logging" to all of them?

Reply:
it shouldn't be necessary - all of the commands should happen in the `logging` namespace

Reply:
It should not be needed

Reply:
hm, it didn't work for me w/o '-n logging' there. Might be that manual execution is different from execution via Jenkins?

Reply:
@t0ffel one of the first things entrypoint.sh does is to `oc project logging` - almost all of the tests use the `oc` command without `-n logging` - if there is a particular place where `oc` requires `-n logging` that is a bug that needs to be fixed asap